onboarding/web-infra.md

```diff
@@ -0,0 +1,23 @@
+# Web Infrastructure Onboarding
+
+This document is an outline of the tasks necessary to onboard a new member of the Web Infrastructure team.
+
+Please note these tasks should only be performed after the person's nomination has passed.
+
+## Tasks
+
+Tasks prefixed by `Member:` are for the new member to complete.
+Tasks prefixed by `Web Admin:` are for a member of `@nodejs/web-admins` to complete.
+
+- [ ] Web Admin: Create an issue in [nodejs/web-team](https://github.com/nodejs/web-team) specifically for onboarding the member and include this list in it.
+  - Please link any additional issues or PRs relating to the member's onboarding to this issue for record keeping.
+- [ ] Web Admin: Recommend member have some form of physical 2FA (i.e. Yubikey) or passkey enabled on their GitHub account, and that any SSH or GPG key attached to their GitHub account have passphrases.
+- [ ] Web Admin: Recommend member sign their Git commits if not already doing so.
+- [ ] Web Admin: Add member to the [@nodejs/web-infra](https://github.com/orgs/nodejs/teams/web-infra) team.
+- [ ] Member: Add yourself to the [membership list document](https://github.com/nodejs/web-team/blob/main/MEMBERS.md#nodejs-web-infra-team-nodejsweb-infra).
+- [ ] Member: Open an issue in [nodejs/admin](https://github.com/nodejs/admin) requesting access to the Web Infra 1Password vault.
+- [ ] Web Admin: Invite member to Node.js' Sentry account.
+  - [ ] Member: Enable physical or passkey 2FA on their Sentry account.
+
+Note there isn't currently a set level of access that Web Infra members has to some resources such as Cloudflare.
```

Comment:
```suggestion
There currently is not a set level of access that @nodejs/web-infra members have on Cloudflare.

Should you need additional access, it may be granted, however, access is not given by default.
```